### PR TITLE
Fixes broken links in sdk docs v5.2.0 - Closes #1220

### LINKED ISSUE #1220: Fix broken links in Lisk protocol and LNS test pages in SDK docs v5.2.0
Fix broken link in What is Lisk protocol page and also in the Lisk Name Service/Chapter 4:Tests page

### Links
https://lisk.com/documentation/lisk-sdk/protocol/index.html#what-is-a-blockchain
https://lisk.com/documentation/lisk-sdk/tutorials/lns/4-tests.html

### PR DESCRIPTION
**Pull request by chris529**
Closes #1220

### CODE REVIEW

File: modules/protocol/pages/index.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 :url_blocks: blocks.adoc
 :url_consensus: consensus-algorithm.adoc
 :url_network: network.adoc
-:url_mainnet: mainnet.adoc
+:url_mainnet: pages/nav.adoc
```
Comment (Tschakki) [inline modules/protocol/pages/index.adoc]: The Mainnet protocol content was moved to the Core docs, so please remove the section to Mainnet related protocol here, or add the corresponding link to the Core protocol docs

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Review by mitsuaki-u — COMMENTED** [review]
LGTM besides @Tschakki comment

File: modules/protocol/pages/index.adoc (hunk 1 above)
Comment (chris529) [reply modules/protocol/pages/index.adoc]: Good point Mona, OK I have now linked it to the Core docs, can you please check this..many thanks 👍

**Review by Tschakki — APPROVED** [review]
Looks good now!

**Pull request merged**